grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/singlequeue/SingleQueueTaskManager.java

```diff
@@ -77,7 +80,12 @@
      */
     public SingleQueueTaskManager(EngineID engineId){
         this.zookeeper = new ZookeeperConnection(client());
-        this.storage = new TaskStateZookeeperStore(zookeeper);
+
+        if(properties.getPropertyAsBool(USE_ZOOKEEPER_STORAGE)){
```

Comment:
Duplicate Code [1]: No chance of putting this somewhere common? Default method? Abstract class?